src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html

```diff
@@ -4,9 +4,7 @@
       Carried out during test
     </dt>
     <dd class="govuk-summary-list__value" id="test-seatbeltInstallationCheckDate">
-      <span *ngIf="testType.seatbeltInstallationCheckDate; else hyphen">
-        {{ testType.seatbeltInstallationCheckDate | date: 'dd/MM/yyyy' }}
-      </span>
+        {{ testType.seatbeltInstallationCheckDate | DefaultNullOrEmpty }}
```

Comment:
should we not format this date field?

Reply:
![image](https://user-images.githubusercontent.com/55185857/82092597-474a4580-9702-11ea-8d22-a6aee9a158b9.png)